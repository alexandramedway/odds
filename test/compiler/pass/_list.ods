do []
do [1]
do [1, 2, 3]
do [42.0, 42.5]
do ["hello", "world"]
do [true, false, true]

do print([])
do print([1])
do print([1, 2, 3])
do print([42.0, 42.5])
do print(["Hello, ", "world!"])
do print([true, false, true])
