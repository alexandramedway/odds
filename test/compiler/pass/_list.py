[]
[1]
[1, 2, 3]
[42., 42.5]
["hello", "world"]
[True, False, True]
print([])
print([1])
print([1, 2, 3])
print([42., 42.5])
print(["Hello, ", "world!"])
print([True, False, True])
