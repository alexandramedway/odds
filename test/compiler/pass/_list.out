[]
[1]
[1, 2, 3]
[42.0, 42.5]
['Hello, ', 'world!']
[True, False, True]
